olm-catalog/serverless-operator/project.yaml

```diff
@@ -28,9 +29,9 @@ requirements:
   golang: '1.19'
   nodejs: 16.x
   ocpVersion:
-    min: '4.8'
-    max: '4.12'
-    label: 'v4.8'
+    min: '4.10'
+    max: '4.13'
```

Comment:
Our schedule is aiming to release after OCP 4.13. That should be fine.

Reply:
@mgencur can you pls confirm that the case? :)

Reply:
@mgencur PTAL if that's correct?

Reply:
Yeah:
Serverless 1.29 GA - 05/24
OCP 4.13 GA - 05/17